MetaMask/metamask-extension
# Moving transaction-alerts component from /ui/pages/confirm-transaction folder to ui/component/app folder

Fixes: #17248 
## Explanation
Moving the component `https://github.com/MetaMask/metamask-extension/tree/develop/ui/pages/confirm-transaction-base/transaction-alerts` to `https://github.com/MetaMask/metamask-extension/tree/develop/ui/components/app`


## Screenshots/Screencaps



### Before



### After



## Manual Testing Steps
1. Start a send transaction and make sure all the alerts are appearing on the confirm transaction page.


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
